### PR TITLE
Pan european/assessment page

### LINKED ISSUE #574: PanEuropean: new page layout

### PR DESCRIPTION
**Pull request by minotogna**
Resolves #574

### CODE REVIEW

File: webapp/main/basePaths.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+// /login/resetPassword
+//
+// #### Assessment
+// /:countryIso/:assessmentType/print/
+// /:countryIso/:assessmentType/print/onlyTables/
+// /:countryIso/:assessmentType/home/
+// /:countryIso/:assessmentType/home/:section/
+// /:countryIso/:assessmentType/:section/
+// /:countryIso/odp/
+// /:countryIso/odp/:tab/
```
Comment (sorja) [inline webapp/main/basePaths.js:25]: Should have /* */ comments

File: webapp/components/countrySelection/components/countryList.js [hunk 2]
```diff
@@ -2,58 +2,27 @@ import './countryList.less'
...
+  const { query } = props
+  return <AssessmentComponent components={Components} query={query} />
 }
 
 CountryList.propTypes = {
-  query: PropTypes.string,
+  query: PropTypes.string.isRequired,
 }
 
 export default CountryList
```
Comment (sorja) [inline webapp/components/countrySelection/components/countryList.js:28]: is this file anymore needed

**Review by sorja — APPROVED** [review]

File: webapp/app/assessment/components/review/reviewIndicator.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import * as R from 'ramda'
 import Icon from '@webapp/components/icon'
 
 import { getIssueSummary, openCommentThread, closeCommentThread } from './actions'
-import { isPrintingMode } from '../print/printAssessment'
+import { isPrintingMode } from '../../../../pages/AssessmentPrint/printAssessment'
```
[the remark this answers is not in the document]
Comment (minotogna) [reply webapp/app/assessment/components/review/reviewIndicator.js:8]: this must be refactored

File: webapp/components/countrySelection/components/countryList.js (hunk 2 above)
Comment (minotogna) [reply webapp/components/countrySelection/components/countryList.js:28]: sure it is

**Pull request merged**